custom/icds_reports/migrations/0173_auto_20200323_1038.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-03-23 10:38
+from __future__ import unicode_literals
+
+import custom.icds_reports.models.aggregate
+
+from django.db import migrations, models
+
+from custom.icds_reports.utils.migrations import (
+    get_composite_primary_key_migrations,
+)
+from corehq.sql_db.operations import RawSQLMigration
+from corehq.sql_db.connections import is_citus_db
+from custom.icds_reports.const import AGG_BIHAR_API_DEMOGRAPHICS
+from custom.icds_reports.utils.migrations import (
+    create_citus_distributed_table
+)
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates', 'database_views'))
+
+
+def _distribute_citus_tables(apps, schema_editor):
+    with schema_editor.connection.cursor() as cursor:
+        if not is_citus_db(cursor):
+            return
+
+        for table, col in [(AGG_BIHAR_API_DEMOGRAPHICS, 'supervisor_id')]:
```

Comment:
This can probably just be
```
create_citus_distributed_table(cursor, AGG_BIHAR_API_DEMOGRAPHICS, 'supervisor_id')
```
no need to iterate over a hardcoded list of length 1